openshift/capabilities-tool-job.yaml

```diff
@@ -40,13 +40,18 @@ spec:
               configMapKeyRef:
                 name: audit-env-var
                 key: bundle_name
+          - name: installMode
+            valueFrom:
+              configMapKeyRef:
+                name: audit-env-var
+                key: installMode
         securityContext:
           privileged: true
         volumeMounts:
         - name: docker-config
-          mountPath: /opt/capabilities-tool/.docker/
+          mountPath: /opt/opcap/.docker/
         - name: kube-config
-          mountPath: /opt/capabilities-tool/.kube/
+          mountPath: /opt/opcap/.kube/
```

Comment:
Just for future reference as Alex mentioned before, you don't want to combine these changes. One is related to renaming while the other is related to adding the flag.

Reply:
Removed the above changes from this commit.